test/model/editions/PublishedIssueTest.scala

```diff
@@ -86,40 +86,90 @@ class PublishedIssueTest extends FreeSpec with Matchers with OptionValues {
+      Some(false),
+      Some(CoverCardImages(
+        mobile = cardImage,
+        tablet = cardImage
+      ))
+    )
+
+    "furniture should be populated when specified, cover cards should be ignored if the media type isn't set to cover card" in {
+      val article = EditionsArticle("123456", 0, Some(articleFurniture))
+      val published = article.toPublishedArticle
+
+      published.furniture shouldBe PublishedFurniture(
         Some("kicker"),
+        Some("headline"),
         Some("trail-text"),
-        Some(true),
-        Some(true),
         Some("byline"),
-        Some("sport-score"),
-        Some(MediaType.Image),
-        None,
-        Some(Image(Some(100), Some(100), "file://image-1.gif", "file://image-1.jpg")),
-        Some(false),
-        None
+        showByline = true,
+        showQuotedHeadline = true,
+        mediaType = PublishedMediaType.Image,
+        imageSrcOverride = Some(PublishedImage(Some(100), Some(100), "file://image-1.jpg")),
+        sportScore = Some("sport-score"),
+        overrideArticleMainMedia = false,
+        coverCardImages = None
       )
-      val article = EditionsArticle("123456", 0, Some(furniture))
+    }
+
+    "furniture should be populated when specified, cover card should be some if media type is covercard " in {
+      val article = EditionsArticle("123456", 0, Some(articleFurniture.copy(mediaType = Some(MediaType.CoverCard))))
       val published = article.toPublishedArticle
 
+      val publishedImage = PublishedImage(Some(100), Some(100), "file://src.jpg")
+
       published.furniture shouldBe PublishedFurniture(
         Some("kicker"),
         Some("headline"),
         Some("trail-text"),
         Some("byline"),
         showByline = true,
         showQuotedHeadline = true,
-        mediaType = PublishedMediaType.Image,
-        imageSrcOverride = Some(PublishedImage(Some(100), Some(100), "file://image-1.jpg")),
+        mediaType = PublishedMediaType.CoverCard,
+        imageSrcOverride = None,
         sportScore = Some("sport-score"),
         overrideArticleMainMedia = false,
-        coverCardImages = None
+        coverCardImages = Some(PublishedCardImage(publishedImage, publishedImage))
       )
     }
+
+    "furniture should throw if set to media type is covercard but we dont have both mobile and tablet " in {
+      val coverCardFurniture = articleFurniture
+        .copy(mediaType = Some(MediaType.CoverCard))
+        .copy(coverCardImages = Some(CoverCardImages(
+          mobile = cardImage,
+          tablet = None
+        )))
+
+      val swapped = coverCardFurniture.copy(coverCardImages = Some(CoverCardImages(
+        mobile = None,
+        tablet = cardImage
+      )))
+
+      List(
+        EditionsArticle("123456", 0, Some(coverCardFurniture)),
+        EditionsArticle("123456", 0, Some(swapped))
+      ).foreach { a =>
+        assertThrows[IllegalStateException] {
+          a.toPublishedArticle
+        }
+      }
+    }
```

Comment:
Instinctively I think we should not prevent exporting in this scenario. I would much prefer to log a `WARN` and then flip `MediaType` to `UseArticleTrail`. Thoughts @akash1810 ?

Reply:
We do validation on the client to prevent this from happening. If this breaks we should probably revert to the default so 24/7 don't have to get called.

Reply:
If we're guarding against this on save (on both client and server) I don't think we'd need to cater for this scenario.

Defaulting to `UseArticleTrail` is the simplest way to ensure we can still publish - we'd want to make this obvious on the client too (the warning message from the PR description goes some way to achieving this).